### PR TITLE
Update on auto-create namespace

### PR DESCRIPTION
**Pull request by MelwynPaul**

### CODE REVIEW

File: content/en/docs/samples/argo-cd.md [hunk 1]
```diff
@@ -48,7 +50,7 @@ This example provides information about how to deploy the `Hello-helidon` applic
 5. Select the required **Sync Policy** option:
    - `Automatic` - By default, every three minutes, Argo CD checks the specified Git repository and synchronizes the updates in Kubernetes to the Git repository.
    - `Manual` - For manually synchronizing the updates to the Git repository, use the **Manual** option.
-6. In the **Sync Options** section, select **Auto-Create Namespace**.
+6. _Optional_: If you want to auto-create a namespace, in the **Sync Options** section, select **Auto-Create Namespace**.
```
Comment (rosemarymarano) [inline content/en/docs/samples/argo-cd.md:53]: I'm glad that you got the clarifying information from Preethi. However, even though step 6, auto-create a namespace is not required, as far as I know, having a labelled namespace is required. 

Maybe we can add info to this step like, "For more information, see the preceding **NOTE** about whether to pre-create or auto-create the namespace." 

Or, add something like Dave's text, "By auto-creating the namespace, the application will be deployed outside of the service mesh. The best practice is to pre-create the namespace and label it." (If you use this suggestion, so as to not be repetitious, you can remove the second sentence of the preceding NOTE.)

The idea being that when users come to this step, even though you have marked it _Optional_, creating a namespace is not optional.

File: content/en/docs/samples/argo-cd.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ The following is a sample procedure to configure a private Git repository throug
 
 To deploy applications in a custom namespace, create Argo CD applications that specify the Git repository path, which Argo CD requires to synchronize and deploy the applications in the specified namespace.
 
+**Note**: You can either pre-create a namespace and label it or auto-create a namespace when deploying an application. If you auto-create a namespace, the application will be deployed outside of the service mesh.
```
Comment (rosemarymarano) [inline content/en/docs/samples/argo-cd.md:42]: -> **NOTE**:

**Review by rosemarymarano — COMMENTED** [review]
Some suggesitons

File: content/en/docs/samples/argo-cd.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/samples/argo-cd.md:53]: @MelwynPaul Also, please remove the italics. I think that calling out the word Optional, is also misleading.

Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md]: - Removed "Optional". As the step starts with a "If" statement
- Went with the second suggestion and added info under Step 6.

Yes, informing the user at the beginning of the procedure (to pre-create a namespace) - and at Step 6, talking about what will happen if we select auto-create namespace covers everything. 👍

File: content/en/docs/samples/argo-cd.md [hunk 3]
```diff
@@ -39,7 +37,7 @@ The following is a sample procedure to configure a private Git repository throug
 
 To deploy applications in a custom namespace, create Argo CD applications that specify the Git repository path, which Argo CD requires to synchronize and deploy the applications in the specified namespace.
 
-**Note**: You can either pre-create a namespace and label it or auto-create a namespace when deploying an application. If you auto-create a namespace, the application will be deployed outside of the service mesh.
+**Note**: You can either pre-create a namespace and label it or auto-create a namespace when deploying an application.
```
Comment (rosemarymarano) [inline content/en/docs/samples/argo-cd.md:40]: Note -> (should be all caps) **NOTE**

Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md]: Yes. Updated now.

File: content/en/docs/samples/argo-cd.md (hunk 2 above)
Comment (MelwynPaul) [reply content/en/docs/samples/argo-cd.md]: Thank you. Fixed it now.

**Review by rosemarymarano — APPROVED** [review]

**Review by preethignanesh — APPROVED** [review]
Looks good

**Pull request merged**